.spelling

```diff
@@ -5,3 +5,8 @@ kickstart
 75mm
 SKL_ADDR
 9am
+StudentRobotics
```

Comment:
Could we prefix this with `@` to be clear that this is our twitter handle rather than a generally used form of our name (which it isn't).

Reply:
Unfortunately that doens't work. The spell-checking already strips out the leading `@`, so it's really only check if "StudentRobotics" is a word (which it's not).

Reply:
And yet somehow it doesn't complain about email addresses? Most odd.
Is there a way to have comments in the spellings file? Not that that's going to prevent false positives. Is there a way to mark a specific word as an ok spelling without adding it to the spellings file?

If it's too complex, this likely isn't worth it.